File: client/src/app/_models/repository.ts
// src/app/_models/repository.model.ts

export interface Repository {
    id: number;
    label: string;       // e.g., "JoVE"
    imageUrl: string;    // e.g., "/assets/jove.png"
    linkUrl: string;     // e.g., "https://www.jove.com/"
}

File: API/Entities/Repository.cs
// API/Entities/Repository.cs
namespace API.Entities
{
    public class Repository
    {
        public int Id { get; set; }
        public string Label { get; set; } = string.Empty;       // e.g., "JoVE"
        public string ImageUrl { get; set; } = string.Empty;    // e.g., "/assets/jove.png"
        public string LinkUrl { get; set; } = string.Empty;     // e.g., "https://www.jove.com/"
    }
}
